### PR TITLE
Android `minuteInterval` prop support

### PR DESCRIPTION
**Pull request by BenderBRodrigez**
# Summary

Based on #66. Fixed Android build for RN 0.62

## Checklist

- [x] I have tested this on a device and a simulator
- [x] I added the documentation in `README.md`
- [ ] I mentioned this change in `CHANGELOG.md`
- [x] I updated the typed files (TS and Flow)
- [x] I added a sample use of the API in the example project (`example/App.js`)

### CODE REVIEW

File: android/src/main/java/com/reactcommunity/rndatetimepicker/RNDismissableTimePickerDialog.java [hunk 1]
```diff
@@ -31,28 +40,240 @@
  *   See: <a href="https://code.google.com/p/android/issues/detail?id=34833">Issue 34833</a>
  * </p>
  */
-public class RNDismissableTimePickerDialog extends TimePickerDialog {
+
+class CustomTimePickerDialog extends TimePickerDialog {
```
Comment (luancurti) [inline android/src/main/java/com/reactcommunity/rndatetimepicker/RNDismissableTimePickerDialog.java:44]: Can you move this class to another java file?

File: android/src/main/java/com/reactcommunity/rndatetimepicker/RNTimePickerDialogFragment.java [hunk 2]
```diff
@@ -55,9 +56,18 @@ static TimePickerDialog getDialog(
...
+      minuteInterval = args.getInt(RNConstants.ARG_INTERVAL);
+    }
+
     RNTimePickerDisplay display = RNTimePickerDisplay.DEFAULT;
     if (args != null && args.getString(RNConstants.ARG_DISPLAY, null) != null) {
-      display = RNTimePickerDisplay.valueOf(args.getString(RNConstants.ARG_DISPLAY).toUpperCase(Locale.US));
+      if (RNMinuteIntervals.isRadialPickerCompatible(minuteInterval)) {
+        display = RNTimePickerDisplay.valueOf(args.getString(RNConstants.ARG_DISPLAY).toUpperCase(Locale.US));
+      } else {
+        display = RNTimePickerDisplay.SPINNER;
```
Comment (luancurti) [inline android/src/main/java/com/reactcommunity/rndatetimepicker/RNTimePickerDialogFragment.java:69]: Why you are setting the display spinner? You can get from `args.getString(RNConstants.ARG_DISPLAY)`

File: android/src/main/java/com/reactcommunity/rndatetimepicker/RNDismissableTimePickerDialog.java (hunk 1 above)
Comment (BenderBRodrigez) [reply android/src/main/java/com/reactcommunity/rndatetimepicker/RNDismissableTimePickerDialog.java]: that's done

File: android/src/main/java/com/reactcommunity/rndatetimepicker/RNTimePickerDialogFragment.java (hunk 2 above)
Comment (BenderBRodrigez) [reply android/src/main/java/com/reactcommunity/rndatetimepicker/RNTimePickerDialogFragment.java:69]: I'm not strong in java, I just fixed some issues from PR #66 to build this and get right necessary functionality in my project, according to the rule "works - don't touch it". In this case I don't understand what are you talking about. I'll be appreciated for more detailed explanation

Comment (luancurti) [reply android/src/main/java/com/reactcommunity/rndatetimepicker/RNTimePickerDialogFragment.java:69]: You are setting the `display` without respect the value of the `display` passed by the user is that right? I think you can check if the user passed the prop checking the `ARG_DISPLAY` if has value you se into `display` if not you set `SPINNER` what do you think?

**Pull request closed without merging**